docs: Loan management module documentation (v13) (#573)
frappe/erpnext_com: erpnext_com/www/docs/user/manual/en/loan-management/loan-security-price.md

<!-- add-breadcrumbs -->
# Loan Security Price
> Introduced in Version 13

**Loan Security Price is the record in which you can log the loan security price and its validity.**

During a loan security price update process, Loan Security Price is used to fetch the valid loan security

> price for updation

Reviewer: ```suggestion
price for updation.
```

To access the Loan Security Price list, go to:
> Home > Loan Management > Loan Security > Loan Security Price

> <img class="screenshot" alt="Loan Security Price" src="{{docs_base_url}}/assets/img/loan-management/loan-security-price.png">

Reviewer: what is the America/New_York label?

Author: It is the timestamp field label added by default based on the region. My test site has USA as country

Reviewer: Why does New_York have an underscore?

## 1. Prerequisites
Before creating and using a Loan Security Price, it is advised that you create the following first:

* [Loan Security Type](/docs/user/manual/en/loan-management/loan-security-type)
* [Loan Security](/docs/user/manual/en/loan-management/loan-security)

## 2. How to Create a Loan Security Price
1. Go to the Loan Security Price list, click on New.

> 2. Select the Loan Security. The loan security type and UOM will be automatically fetched from the Loan Security Type master.

Reviewer: ```suggestion
2. Select the Loan Security. The Loan Security Type and UOM will be automatically fetched from the Loan Security Type master.
```

> 3. Enter loan security price.

Reviewer: ```suggestion
3. Enter Loan Security Price.
```

> 4. Enter the valid from and valid upto timestamps.

Reviewer: ```suggestion
4. Enter the Valid From and Valid Up to timestamps.
```
Up to is never a single word, correct in doctype also
